### PR TITLE
IfcGeotechTypicalSection => IfcTunnelTypicalSection

### LINKED ISSUE #482: [IFC Tunnel]IfcLinearStructureElement rename IfcGeotechnicalTypicalSection to IfcTunnelTypicalSection
Hi Lars,
as discussed today we propose to change the Entity IfcGeotechTypicalSection https://bsi-infraroom.github.io/IFC-Documentation-Tunnel/4_4_0_0/general/HTML/ to cover the same concept more generally for tunnel design, and call it e.g. IfcTunnelTypicalSection.
PDTs can then cover both Geotechnics, tunnel design and related risks, as described earlier e.g. in image below.
PDT "GeotechTypicalSection"
PDT "ExcavationSupportSection"
PDT "RiskSection"

This would allow to apply the concept to many other design-relevant aspects and create user defined PDTs

image

_Originally posted by @j-weil in https://github.com/bSI-InfraRoom/IFC-Specification/issues/447#issuecomment-1285225031_

### PR DESCRIPTION
**Pull request by larswik**
Fixes #482. Also moved to IfcTunnel domain from IfcSjharedInfrastructureElements

### CODE REVIEW

File: IFC4x3/Constants/g/GEOTECHTYPICALSECTION_1V8YeK3DXAxwY3SVvMcwhP.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="GEOTECHTYPICALSECTION_1V8YeK3DXAxwY3SVvMcwhP" Name="GEOTECHTYPICALSECTION" UniqueId="5f222a14-0cd8-4aef-a883-71fe569baad9">
+	<Documentation>Interval along the alignment/built structure with similar ground conditions, as part of the GeotechSynthesis model that represents the connection between the ground model and the built structure. Includes key-properties like expected distribution of ground types (reference to GeotechUnits) and baseline-definition of expected ground conditions and potential hazards, and may also include key-information on design like excavation measures, distribution of support types etc. These properties can be provided using pre- or user-defined property sets, using _IfcRelDefinesByProperties_ or external datasets using _IFcRelAssociatesDataset_.</Documentation>
```
Comment (pjanck) [inline IFC4x3/Constants/g/GEOTECHTYPICALSECTION_1V8YeK3DXAxwY3SVvMcwhP.xml]:
```suggestion
	<Documentation>Interval along the alignment/built structure with similar ground conditions, as part of the GeotechSynthesis model that represents the connection between the ground model and the built structure. Includes key-properties like expected distribution of ground types (reference to GeotechUnits) and baseline-definition of expected ground conditions and potential hazards, and may also include key-information on design like excavation measures, distribution of support types etc. These properties can be provided using pre- or user-defined property sets, using _IfcRelDefinesByProperties_ or external datasets using _IfcRelAssociatesDataset_.</Documentation>
```

**Review by pjanck — APPROVED** [review]
lftm

File: IFC4x3/Constants/g/GEOTECHTYPICALSECTION_1V8YeK3DXAxwY3SVvMcwhP.xml (hunk 1 above)
Comment (SergejMuhic) [reply IFC4x3/Constants/g/GEOTECHTYPICALSECTION_1V8YeK3DXAxwY3SVvMcwhP.xml]: What was the change here? I cannot pinpoint it.

**Review by SergejMuhic — COMMENTED** [review]
Why does GEOTECHTYPICALSECTION have such an elaborate description compared to RISKSECTION? It seems that RISKSECTION does not allow properties?

**Comment by larswik** [comment]
> does not allow properties?

Would you like to remove text from GEOTECHTYPICALSECTION or add text to the others?

**Comment by larswik** [comment]
New proposal for PDT:s
- EXCAVATIONSUPPORT
- RISK
- GEOTECH

**Review by pjanck — APPROVED** [review]
lftm

File: IFC4x3/Constants/g/GEOTECHTYPICALSECTION_1V8YeK3DXAxwY3SVvMcwhP.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="GEOTECHTYPICALSECTION_1V8YeK3DXAxwY3SVvMcwhP" Name="GEOTECHTYPICALSECTION" UniqueId="5f222a14-0cd8-4aef-a883-71fe569baad9">
+	<Documentation>Interval along the alignment/built structure with similar ground conditions, as part of the GeotechSynthesis model that represents the connection between the ground model and the built structure. Includes key-properties like expected distribution of ground types (reference to GeotechUnits) and baseline-definition of expected ground conditions and potential hazards, and may also include key-information on design like excavation measures, distribution of support types etc. These properties can be provided using pre- or user-defined property sets, using _IfcRelDefinesByProperties_ or external datasets using _IfcRelAssociatesDataset_.</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/Constants/g/GEOTECHTYPICALSECTION_1V8YeK3DXAxwY3SVvMcwhP.xml]: GeotechSynthesis and GeotechUnits are apparently concepts of some sort. There are no references to them. What do they mean?

**Review by SergejMuhic — CHANGES_REQUESTED** [review]

File: IFC4x3/Constants/g/GEOTECHTYPICALSECTION_1V8YeK3DXAxwY3SVvMcwhP.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/Constants/g/GEOTECHTYPICALSECTION_1V8YeK3DXAxwY3SVvMcwhP.xml]: I think that, to be sure, we will need to go back to the domain team. There is an IfcGeoScienceModel/GEOTECHSYNTHESISMODEL and an IfcGeoScienceFeature/GEOTECHUNIT, but are they really meaning this in a strict sense?

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**